source/main/gui/panels/GUI_MainSelector.cpp

```diff
@@ -887,7 +888,17 @@ void CLASS::SetPreviewImage(Ogre::String texture)
 void CLASS::ResizePreviewImage()
 {
 	MyGUI::IntSize imgSize(0, 0);
-	Ogre::TexturePtr t = Ogre::TextureManager::getSingleton().load(m_preview_image_texture, Ogre::ResourceGroupManager::AUTODETECT_RESOURCE_GROUP_NAME);
+	Ogre::TexturePtr t;
+	try
+	{
+		t = Ogre::TextureManager::getSingleton().load(m_preview_image_texture, Ogre::ResourceGroupManager::AUTODETECT_RESOURCE_GROUP_NAME);
+	}
+	catch (...)
+	{
+		Ogre::LogManager::getSingleton().stream() << "[RoR|SelectorGUI] Failed to load preview image: " << m_preview_image_texture;
+		m_preview_image_texture.clear();
+		m_Preview->setVisible(false);
+	}
```

Comment:
If we catch the exception inside of `ResizePreviewImage()`, won't we then run into issues here:
https://github.com/RigsOfRods/rigs-of-rods/blob/a484facaddbcafe0fa71427c33dd08cb5a731c10/source/main/gui/panels/GUI_MainSelector.cpp#L877-L879